tests/state/test_get_list_state.py

```diff
@@ -0,0 +1,365 @@
+"""
+Provide tests for command line interface's get a list of states command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+ADDRESS_WITH_STATE = '0000000000000000000000000000000000000000000000000000000000000000000001'
+
+
+def test_get_list_state():
+    """
+    Case: get a list of states.
+    Expect: states are returned.
```

Comment:
```suggestion
    Expect: list of states is returned.
```
Change in each of the test cases.

Reply:
@anastasiia-bilova, done.